articles/azure-maps/tutorial-load-geojson-file-android.md

```diff
@@ -55,7 +57,7 @@ map.sources.add(source);
 
 ::: zone pivot="programming-language-kotlin"
 
-4. Go into the **MainActivity.kt** file and add the following code inside the callback for the `mapControl.onReady` event, inside the `onCreate` method. This code loads the **SamplePoiDataSet.json** file from the assets folder into a data source using `importDataFromUrl` method and then adds it to the map.
+1. Go into the **MainActivity.kt** file and add the following code inside the callback for the `mapControl.onReady` event, inside the `onCreate` method. This code loads the **SamplePoiDataSet.json** file from the assets folder into a data source using `importDataFromUrl` method and then adds it to the map.
```

Comment:
Should we also use _MaiActivity.kt_ and _SamplePoiDataSet.json_ here? (And also line 140, 153)